app/models/subscriber.rb

```diff
@@ -10,4 +10,5 @@
 #
 
 class Subscriber < ActiveRecord::Base
+  has_one :subscriber_datum
```

Comment:
`"metadata".singularize == "metadatum"` though I'll hack it in there cause it just makes more sense